plan/relations.go

```diff
@@ -516,6 +524,9 @@ func (p *ProjectRel) RecordType() types.StructType {
 		Types:       output,
 	}
 }
+func (p *ProjectRel) RecordType() types.StructType {
+	return p.Remap(p.directOutputSchema())
```

Comment:
Can we unexport `remap()` as part of this change. We've removed it from the interface but I think we should also remove from export.

Reply:
Unexported.